### PR TITLE
Add documentation about docker container

### PR DESCRIPTION
**Pull request by tbouron**

### CODE REVIEW

File: docs/swarm-cluster.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ We interact directly with the swarm manager cluster's load balancer as if it wer
 The load-balancer will redirect traffic to a healthy manager when a manager fails.  The replacer policy will detect the failure and replace the failed manager.
 
 #### A cluster of swarm nodes
-These nodes are where docker containers are deployed to. The cluster has an AutoScalerPolicy and will scale up due to high CPU usage.
+These nodes are where [docker containers](docker-engine.html) are deployed to. The cluster has an AutoScalerPolicy and will scale up due to high CPU usage.
```
Comment (drigodwin) [inline docs/swarm-cluster.md]: I would remove the 'to' from this first sentance.

File: docs/docker-engine.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+---
+layout: base
+title: Docker Engine Documentation
+---
+
+### Overview
+Clocker provides a base Docker Engine entity that will deploy a Docker Engine on any location and run a Docker image automatically (if required).
```
Comment (drigodwin) [inline docs/docker-engine.md]: I would say any cloud / VM location.

**Comment by tbouron** [comment]
@drigodwin as it turns out, I didn't document the right entity. Should be fine now

File: docs/swarm-cluster.md (hunk 1 above)
Comment (m4rkmckenna) [inline docs/swarm-cluster.md]: `and will scale up due to high CPU usage` ➡️  `that will scale up under high CPU usage`

File: docs/docker-container.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+---
+layout: base
+title: Docker Container Documentation
+---
+
+### Overview
+Clocker provides a base Docker container entity that will deploy a Docker Engine on any cloud / VM location. and run a Docker image automatically into a Docker container.
```
Comment (drigodwin) [inline docs/docker-container.md]: Clocker provides a base Docker container entity that will deploy a Docker Engine and then start a selected Docker image automatically in it's Docker container. This entity can be deployed in any cloud / VM location.

File: docs/docker-container.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+The Docker container entity comes with built-in configuration that allows you to control how your Docker container will be deployed and managed.
+
+| Config Name                 | Description                                                                                    |
+|-----------------------------|------------------------------------------------------------------------------------------------|
+| docker.image                | The docker image to use when running the container                                             |
+| docker.run.arguments        | Arguments to pass to the docker run command                                                    |
+| docker.run.volumes          | List of volumes to mount. Items follow the documented docker format for the '-v' option        |
+| docker.run.env              | Map of environment variables to pass to the container                                          |
+| docker.restart              | Restart policy on the container. One of no, on-failure[:max-retries], always or unless-stopped |
+| image.run.additionaloptions | Additional options to pass to the 'docker run' command                                         |
```
Comment (drigodwin) [inline docs/docker-container.md]: What kind of additional options? link to the docker docs?

**Pull request merged**